src/app/pages/private/profile/profile.component.html

```diff
@@ -28,17 +28,43 @@ <h4>Profile</h4>
             <form-input inputType="email" name="email" placeholder="hello@example.com" readonly="true"></form-input>
 
             <label>Mobile Number</label>
-            <form-input inputType="text" name="mobile" placeholder="9884098840" readonly="true"></form-input>
+            <form-input inputType="text" name="mobile" placeholder="9884098840"></form-input>
 
             <div class="form-group">
               <label>2FA Enabled</label><br>
               <ui-switch (click)="updateTwoFactor()" formControlName="two_fa_enabled"></ui-switch>
             </div>
-
             <input [disabled]="!this.profileForm.valid" class="btn btn-primary" type="submit" value="Update Profile">
           </form>
         </div>
       </div>
+
+      <ng-template #content let-modal>
+        <form (ngSubmit)="updateMobile()" [formGroup]="otpForm" class="otp p-2">
```

Comment:
1. Can we have form tag inside modal-body?
2. otpForm -> can have better detailed name